lib/build_runner.zig

```diff
@@ -369,8 +375,14 @@ fn runStepNames(
     }
 
     const starting_steps = try arena.dupe(*Step, step_stack.keys());
+
+    const useed = std.hash.XxHash64.hash(0x1312, b.seed);
+    var rng = std.rand.DefaultPrng.init(useed);
```

Comment:
how about instead accepting a 32-bit integer, possibly hex encoded? This is fundamentally simpler, and does not leave the user wondering what string they should use for the seed.

Reply:
This is what I was going for at first, but I figured it would just be simpler for the user to accept any kind of data as argument, since you can't have bad formatting and your source of (random) seed can be pretty much anything (time[date] in any format, user or machine name, ...).
I'd expect the typical seed to be a number, whatever its format, but it does not have to be a hard requirement and the additional complexity is arguably very limited (a single hash over the bytes).

What do you think ?

Reply:
I think that hashing the seed is unnecessary. [I Want Simple, Not Just Easy](https://kristoff.it/blog/simple-not-just-easy/).